frontend/package.json

```diff
@@ -65,6 +65,8 @@
     "cypress": "^10.3.0",
     "cypress-fail-fast": "^5.0.0",
     "cypress-file-upload": "^5.0.2",
+    "eslint": "^8.24.0",
```

Comment:
do we really need a linter ?

Reply:
Yes, we should be using a linter in the frontend code.